code/datums/brain_damage/magic.dm

```diff
@@ -124,3 +124,9 @@
 	desc = "It's coming closer..."
 	image_icon = 'icons/mob/simple/lavaland/lavaland_monsters.dmi'
 	image_state = "curseblob"
+	var/datum/brain_trauma/magic/stalker_multiple/host
+
+/obj/effect/client_image_holder/stalker_phantom/Destroy(force)
+	. = ..()
+	host.stalkers -= src
+	host = null
```

Comment:
```suggestion
/obj/effect/client_image_holder/stalker_phantom/Destroy(force)
	if(host)
		host.stalkers -= src
		host = null
	return ..()
```

I'm seeing this runtime pop up a lot in grafana: 
![image](https://github.com/user-attachments/assets/baece33f-0759-4919-9d4b-d8f5710e8210)